doc/source/data/api/data_representations.rst

```diff
@@ -3,54 +3,53 @@
 Data Representations
 ====================
 
+.. currentmodule:: ray.data
+
 .. _block-api:
 
 Block API
 ---------
 
-.. autoclass:: ray.data.block.Block
-
-.. autoclass:: ray.data.block.BlockExecStats
-    :members:
-
-.. autoclass:: ray.data.block.BlockMetadata
-    :members:
-
-.. autoclass:: ray.data.block.BlockAccessor
-    :members:
+.. autosummary::
+   :toctree: doc/
```

Comment:
This `:toctree: doc/` forces the individual API to be generated in separate page. with `.../doc/<API-name>.html` link.

Example:

https://ray--31204.org.readthedocs.build/en/31204/data/api/doc/ray.data.Dataset.map_batches.html